inst/Upgrades.qml

```diff
@@ -12,4 +12,67 @@ Upgrades
 
 		ChangeRename { from: "conditioningVariables";	to: "partialOutVariables" }
 	}
+
+	// option renaming for syntax
+	Upgrade
+	{
+		functionName:	"Correlation"
+		fromVersion:	"0.16.3"
```

Comment:
On github the tab seems to be a bit close. If this is okay in QML then it's fine. Just wondering.